lib/ui/onboarding_screens/screen3.dart

```diff
@@ -0,0 +1,253 @@
+    'Move your finger on the image to create a spray can pattern.',
+    'Position the cursor where you want to draw. Tap to activate the cursor. Move your finger to draw. Tap again to deactivate.',
+    'Write text and format it. Resize the text box afterwards. Tap on the checkmark to insert the text on the image.',
+    'Move and resize the rectangle to cover the area you want to stamp. Tap on copy or cut to select the area. Move it, then tap on paste to stamp.',
+    'Use to transform the image.',
+    'Import an image from the gallery to the stamp tool.',
+    'Tap on the image to select a color.',
+    'Similar to the brush tool with a watercolor effect. However you can also change the strength of the brush with the slider in the color menu.',
+    'Move your finger on the image on different drawings to smudge them.',
+    'Mark area which should not be erased.',
+  ];
+
+  var title = Row(
+    children: const [
+      Text(
+        'Tools',
+        style: TextStyle(
+          color: Colors.white,
+          fontSize: 24,
+        ),
+        textAlign: TextAlign.start,
+      )
+    ],
+  );
+
+  Icon? toolIcon;
+
+  var desc = const Text(
+    'Select the tool you want to use.',
+    style: TextStyle(
+      color: Colors.white,
+      fontSize: 15,
+    ),
+    textAlign: TextAlign.start,
+  );
+
+  void toolPressed(int i) {
+    setState(
+      () {
+        title = Row(
+          children: [
+            Text(
+              titles[i],
+              style: const TextStyle(
+                color: Colors.white,
+                fontSize: 24,
+              ),
+              textAlign: TextAlign.start,
+            ),
+            Container(
+              padding: const EdgeInsets.only(left: 50),
+              child: SvgPicture.asset(
+                icons[i],
+                height: 24,
+                color: Theme.of(context).colorScheme.onSurface,
+              ),
+            ),
+          ],
+        );
+
+        desc = Text(
+          descriptions[i],
+          style: const TextStyle(
+            color: Colors.white,
+            fontSize: 15,
+          ),
+          textAlign: TextAlign.start,
+        );
+      },
+    );
+  }
+
+  void brush() => toolPressed(0);
+
+  void hand() => toolPressed(1);
+
+  void eraser() => toolPressed(2);
+
+  void line() => toolPressed(3);
+
+  void shapes() => toolPressed(4);
+
+  void fill() => toolPressed(5);
+
+  void sprayCan() => toolPressed(6);
+
+  void cursor() => toolPressed(7);
+
+  void text() => toolPressed(8);
+
+  void stamp() => toolPressed(9);
+
+  void transform() => toolPressed(10);
+
+  void importImage() => toolPressed(11);
+
+  void pipette() => toolPressed(12);
+
+  void watercolor() => toolPressed(13);
+
+  void smudge() => toolPressed(14);
+
+  void clipArea() => toolPressed(15);
+
+  List<BottomNavigationBarItem> _getBottomNavigationBarItems(int l, int r) {
+    List<BottomNavigationBarItem> items = [];
+    for (int i = l; i <= r; i++) {
```

Comment:
I think this would be more flutter-like if we used a ListViewbuilder

Reply:
I think ListViewbuilder will be for creating a ListView, rather we can use the List.generate() method

Reply:
Okay, then lets try that :)